content/docs/start/data-pipelines.md

```diff
@@ -200,10 +202,10 @@ stages:
 
 <details>
 
-### ⚙️ Expand to add more stages.
+### ⚙️ Expand to add more stages ☰
```

Comment:
Idk actually... It already has an icon. + not v clear why 3 lines apply to adding stages (stacking?)

Reply:
BTW this applies to all the previous similar changes 😅 Should we keep either ⚙️ OR some other emoji but just one per expanding title?

![image](https://user-images.githubusercontent.com/1477535/114668283-2ac5dd00-9cc6-11eb-8168-71819fd110f9.png)
![image](https://user-images.githubusercontent.com/1477535/114668311-34e7db80-9cc6-11eb-9f58-ce82942ccd58.png)

Reply:
idk - felt bad about removing the :bomb: emoji so thought the article was begging for more. Personally I like consistency so

1. must continue using :gear: and :bulb: using current conventions
2. maybe also use other emojis (but not as replacements)

I don't like the extra emojis at all but in terms of keeping the tone light & engaging suppose they're ok. Would make me v. happy to remove all of them.

Reply:
If those are the only options I vote for 1. I get the consistency argument but at the same don't see why ⚙️ relates to expanding a section (that's more like, "configure something"). I think expandable sections already look consistent once rendered (having the cursor icon to the right) so I could vote go for 3. replace gears for different emojis (maybe at the end of the sentence?). Up to you though

> (4) Would make me v. happy to remove all of them.

We could discuss, please open an issue ⚔️

Reply:
removed extra emojis!